scm_source/api.py

```diff
@@ -15,6 +15,8 @@ def get_scm_source_data(author: str):
     status = get_output(['git', 'status', '--porcelain'])
     if status:
         rev = '{} (locally modified)'.format(rev)
+    if author is None:
+        author = 'UNDEFINED'
     data = {'url': 'git:{}'.format(url), 'revision': rev, 'author': author, 'status': status}
```

Comment:
I think the empty string has the same meaning as "null" here (i.e. we don't know the author) -> replacing it with "unknown" is fine IMO